soccer/src/soccer/strategy/agent/position/runner.cpp

```diff
@@ -0,0 +1,61 @@
+#include "runner.hpp"
+
+#include <spdlog/spdlog.h>
+
+namespace strategy {
+
+Runner::Runner(int r_id) : Position(r_id) {
```

Comment:
```suggestion
Runner::Runner(int r_id) : Position(r_id), current_state_{LEFT_SIDE} {
```
this is a little more idiomatic for setting default member values.

Reply:
alternatively, feel free to write a default value into the declaration in the header